doc/strace.1.in

```diff
@@ -1010,6 +1011,15 @@ Print the syscall number.
 .if '@USE_LIBDW_FALSE@'#' Print the execution stack trace and source code information of the traced
 .if '@USE_LIBDW_FALSE@'#' processes after each system call. This option expects the target program is compiled
 .if '@USE_LIBDW_FALSE@'#' with appropriate debug options: "\-g" (gcc), or "\-g \-gdwarf-aranges" (clang).
+.if '@ENABLE_STACKTRACE_FALSE@'#' .TP
+.if '@ENABLE_STACKTRACE_FALSE@'#' .B \-\-stacktrace\-frames\-limit = limit
+.if '@ENABLE_STACKTRACE_FALSE@'#' Print no more than this amount of stack trace frames
```

Comment:
I suggest mentioning the default limit here as well, similar to the way the maximal string size is mentioned in the description of --string-limit.

Reply:
Done